tools/test/audio/comp_run.sh

```diff
@@ -5,14 +5,114 @@
 # stop on most errors
 set -e
 
-COMP=$1
-DIRECTION=$2
-BITS_IN=$3
-BITS_OUT=$4
-FS1=$5
-FS2=$6
-FN_IN=$7
-FN_OUT=$8
+usage ()
+{
+    cat <<EOFHELP
+Usage:     $0 <options> <comp direction bits_in bits_out fs_in fs_out input output>
+Example 1: $0 volume playback 16 16 48000 48000 input.raw output.raw
```

Comment:
Add some `-e` or other option in this example to make it even clearer that combining flags and positional parameters is OK.